src/types/region.ts

```diff
@@ -0,0 +1,11 @@
+// noinspection JSUnusedGlobalSymbols
+
+export enum Region {
+  Eu = 'EU',
+  HomeCountry = 'Home Country',
```

Comment:
dit heet in de api Domestic vgm

Reply:
O bij carrier_price krijgen we `Home Country` terug